src/scripts/generator_base.py

```diff
@@ -69,10 +69,7 @@ def create_false_numbers_array_string(self, false_values_count):
     def create_declarations_string(self, declarations_dictionary):
         declaration_string = ""
         for key, value in declarations_dictionary.items():  # For E.g.: a0 = 38473837;
-             if isinstance (value, str): # Quote string values
-                 declaration_string += "%s = '%s';\n" % (str(key), str(value))
-             else:
-                 declaration_string += "%s = %d;\n" % (str(key), value)
+             declaration_string += "%s = %s;\n" % (str(key), str(value))
```

Comment:
generate_binary_ops.py, src/scripts/generator_base.py, src/scripts/generate-function-call.py both uses this function. Could this change affect the other generator scripts behavior? Please check it.